docs/sources/administration/api-keys/create-api-key.md

```diff
@@ -14,13 +14,13 @@ weight: 50
 
 Create an API key when you want to manage your computed workload with a user.
 
-For more information about API keys, refer to [About API keys in Grafana]({{< relref "about-api-keys.md" >}}).
+For more information about API keys, refer to [About API keys in Grafana]({{< relref "about-api-keys/" >}}).
 
-This topic shows you how to create an API key using the Grafana UI. You can also create an API key using the Grafana HTTP API. For more information about creating API keys via the API, refer to [Create API key via API]({{< relref "../../developers/http_api/create-api-tokens-for-org.md#how-to-create-a-new-organization-and-an-api-token" >}}).
+This topic shows you how to create an API key using the Grafana UI. You can also create an API key using the Grafana HTTP API. For more information about creating API keys via the API, refer to [Create API key via API]({{< relref "../../developers/http_api/create-api-tokens-for-org/#how-to-create-a-new-organization-and-an-api-token" >}}).
```

Comment:
@jdbaldry, is this target going to work properly?

Reply:
(And similar ones like it that have `#`...) I assume so because that index file gets resolved via Hugo. Has this been tested?

Reply:
Yeah relrefs have been tested to not contain broken links. However, I can't say for sure that all page anchors (the bit after the `#`) have been tested. Hugo doesn't check them automatically and we need the Mimir doc-validator tool in place to do that.